examples/docs_snippets/docs_snippets/guides/components/integrations/powerbi-component/10-launch.txt

```diff
@@ -0,0 +1,26 @@
+dg launch --assets '*'
+
+/Users/ben/repos/dagster/python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/antlr_asset_selection.py:92: BetaWarning: Parameter `include_sources` of function `AssetSelection.all` is currently in beta, and may have breaking changes in minor version releases, with behavior changes in patch releases.
+  return AssetSelection.all(include_sources=self.include_sources)
+2025-05-22 10:08:53 -0700 - dagster - DEBUG - __ASSET_JOB - 123e4567-e89b-12d3-a456-426614174000 - 31984 - RUN_START - Started execution of run for "__ASSET_JOB".
```

Comment:
idk if this all is necessary to show in the output -- i do appreciate the craftsmanship in making this snapshot stable and not mess up git history and all that, but I feel like we could get away with just showing the command invocation without the output in the docs, and separately testing that the command succeeds.